content/blog/content/blog/2024-11-14-prometheus-3-0.md

```diff
@@ -0,0 +1,123 @@
+
+> NOTE: While “NoUTF8EscapingWithSuffixes” strategy allows special characters, it still adds required suffixes for the best experience. See [the proposal on the future work to enable no suffixes](https://github.com/prometheus/proposals/pull/39) in Prometheus.
+
+## Native Histograms
+
+Native histograms are a Prometheus metric type that offer a higher efficiency and lower cost alternative to Classic Histograms. Rather than having to choose (and potentially have to update) bucket boundaries based on the data set, native histograms have pre-set bucket boundaries based on exponential growth.
+
+Native Histograms are still experimental and not yet enabled by default, and can be turned on by passing `--enable-feature=native-histograms`. Some aspects of Native Histograms, like the text format and accessor functions / operators are still under active design.
+
+## Breaking Changes
+
+The Prometheus community strives to [not break existing features within a major release](https://prometheus.io/docs/prometheus/latest/stability/). With a new major release we took the opportunity to clean up a few, but small, long-standing issues. In other words, Prometheus 3.0 contains a few breaking changes. This includes changes to feature flags, configuration files, PromQL, and scrape protocols.
+
+Please read the [migration guide](https://prometheus.io/docs/prometheus/3.0/migration/) to find out if your setup is affected and what actions to take.
+
+# Performance
+
+It’s impressive to see what we have accomplished in the community since Prometheus 2.0. We all love numbers, so let’s celebrate the efficiency improvements we made for both CPU and memory use for the TSDB mode. Below you can see performance numbers between 3 Prometheus versions on the node with 8 CPU and 49 GB allocatable memory.
+
+* 2.0.0 (7 years ago)
+* 2.18.0 (4 years ago)
+* 3.0.0 (now)
+
+![Memory bytes](/static/blog/2024-09-24/memory_bytes_ui.png)
+
+![CPU seconds](/static/blog/2024-09-24/cpu_seconds.png)
+
+It’s furthermore impressive that those numbers were taken using our [prombench macrobenchmark](https://github.com/prometheus/prometheus/pull/15366) 
+that uses the same PromQL queries, configuration and environment–highlighting backward compatibility and stability for the core features, even with 3.0.
+
+# What's Next
+
+There are still tons of exciting features and improvements we can make in Prometheus and the ecosystem. Here is a non-exhaustive list to get you excited and… 
+hopefully motivate you to contribute and join us!
+
+* New, more inclusive **governance**
+* More **OpenTelemetry** compatibility and features
+* OpenMetrics 2.0, now under Prometheus governance!
+* Native Histograms stability (and with custom buckets!)
+* More optimizations!
+* UTF-8 support coverage in more SDKs and tools
+
+# Try It Out!
```

Comment:
can we put "try it out" at the top of the post? I think some percentage of people would like to download first and then read details later

Reply:
I tried moving the `Try It Out` section to the top after the intro, but the box that has the links for the `What's New` section now appear in the `Try It Out` section... maybe in the intro we can include a link to download it like we did in the 3.0 beta post, then leave the `Try It Out` at the bottom?
<img width="1349" alt="Screenshot 2024-11-05 at 6 10 05 PM" src="https://github.com/user-attachments/assets/3b81462c-5465-4978-8247-023fe0ba5d93">

Reply:
I did try adding the `Try It Out` section to the top below the intro, but it also puts the links for `What's New` next to it...
![Screenshot 2024-11-13 at 11 50 10 AM](https://github.com/user-attachments/assets/6d094084-e64a-497e-8c4c-888d5f4b3710)

Reply:
Maybe we can add a link to download it in the intro, like we did in the 3.0 Beta post, and keep the `Try It Out` section at the bottom?

Reply:
I don't mind having the `Try it out` section further down. People should read the whole blog post before trying it out :) 
But I don't feel strongly about this

Reply:
I like this order